lib/Net/Async/Github.pm

```diff
@@ -247,6 +247,83 @@ sub Net::Async::Github::Repository::grant_team {
     )
 }
 
+=head2 create_branch
+
+Creates a new branch.
+
+Takes the following named parameters:
+
+=over 4
+
+=item * C<owner> - which organisation owns the target repository
+
+=item * C<repo> - the repository to raise the PR against
+
+=item * C<branch> - new branch name that will be created
+
+=item * C<sha> - the SHA1 value for this branch
```

Comment:
we need to  send a sha value to create a branch?

Reply:
Yes. a branch in fact is a reference, which will point to one commit. This SHA1 value is that commit's

Reply:
Here is an article http://www.levibotelho.com/development/commit-a-file-with-the-github-api/